website/src/pages/index.tsx

```diff
@@ -88,9 +87,9 @@ export default function Index() {
         },
       },
       {
-        title: 'Handle file uploads with GraphQL?',
+        title: 'Handle file uploads with GraphQL',
         description:
-          'Simply enable uploads and start accepting files for GraphQL mutations.',
+          'Simply enable uploads and start accepting files for GraphQL mutations. No additional packages needed!',
```

Comment:
```suggestion
          'Enable file uploads with GraphQL mutations with no additional packages needed.',
```

How about this instead?